.eslintrc

```diff
@@ -21,8 +21,11 @@
     "airbnb"
   ],
   "rules": {
-    "arrow-parens": "warn",
-    "implicit-arrow-linebreak": "warn",
+    "arrow-parens": "error",
+    "implicit-arrow-linebreak": "off",
```

Comment:
I think this only one that I am not sure about, personally I'd prefer it to be `["error", "beside"]`, but we can discuss it on slack or in the ticket.